python/cm/upgrade.py

```diff
@@ -256,6 +256,23 @@ def rpm_cmp_reverse(obj1, obj2):
         return res
 
 
+def re_count_issue(obj, upgrade):
+    """Re-count of issues from the bottom up"""
```

Comment:
Or? It will recount from bottom for cluster and provider only. May be it is better to mark this in docstring? Besides it would be nice to have recount for service too, just in case.

Reply:
Fixed

Reply:
Not fixed, see new comment